plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -38,12 +41,105 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
-abstract class BaseMySqlIntegrationSmokeTest
-        // TODO extend BaseConnectorTest
-        extends AbstractTestIntegrationSmokeTest
+abstract class BaseMySqlConnectorTest
+        // TODO(https://github.com/trinodb/trino/issues/7019) define shorter tests set that exercises various read and write scenarios (a.k.a. "a smoke test")
```

Comment:
TODO is in the wrong place. You need one full test suite and one smoke test for `TestGlobalTransactionMySqlConnectorTest`.

Reply:
Oh - I guess I read the intention wrongly, and assumed we want both smoke and full tests, so we can run this or this depending on the situation.